Documentation/project-docs/event-logging-guide.md

```diff
@@ -0,0 +1,25 @@
+#Event Logging in CoreCLR
+
+For event logging mechanism in coreclr, we are using the [ETW](https://msdn.microsoft.com/en-us/library/windows/desktop/bb968803%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)- Event Tracing for Windows as a template for all logging mechanism
+
+## How to Add Events to the Runtime:
+
+**Step1**: Edit the [manifest] (https://github.com/dotnet/coreclr/blob/master/src/vm/ClrEtwAll.man). For guidelines look at the existing events in the manifest and this guide for [ETW Manifests](https://msdn.microsoft.com/en-us/library/dd996930%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)
+
+**Step 2**:
+
+-*For Windows*: TBD
+
+-*For Linux*:
+Run the script at [<Root>/src/inc/genXplatEtw.pl](https://github.com/dotnet/coreclr/blob/master/src/inc/genXplatEtw.pl)
+
+This will modify all the files involved in generating the code for the underlying Event logging system
```

Comment:
It would be good to list the files so that developers can ensure their changes are complete.  Also, perhaps mention the technology being used for each platform along with the instructions as well.